### PR TITLE
jQuery dependency removed

### PR DESCRIPTION
**Pull request by frikeldon**
Hi
I removed jquery dependency from knockout-dragdrop.
If you are not interested in this change, sorry about the waste of time, and if you want some specific change before accepting the pull-request, tell me, and i'll be glad make it.

PD: I test it in Chrome, Firefox, Explorer 9 and Explorer 11 (it's not compatible with Explorer 8)

### CODE REVIEW

File: lib/knockout.dragdrop.js [hunk 1]
```diff
@@ -418,27 +471,27 @@
...
+                        overlay.style.bottom = 0;
+                        overlay.style.cursor = 'move';
+                        overlay.style.backgroundColor = 'white';
+                        overlay.style.opacity = 0;
+                        overlay.style.filter = 'alpha(opacity=0)';
+                        overlay.style.userSelect = 'none';
+                        overlay.style.webkitUserSelect = 'none';
+                        overlay.style.MozUserSelect = 'none';
+                        overlay.style.msUserSelect = 'none';
+                        overlay.style['-o-user-select'] = 'none';
```
Comment (papandreou) [inline lib/knockout.dragdrop.js]: `overlay.style.OUserSelect = 'none';`

**Comment by frikeldon** [comment]
I corrected it. Thanks for the note, I wasn't able to find the property name.

**Comment by sunesimonsen** [comment]
Thank you so much for the hard work. I tested it a bit in different browsers and it seems to work fine in IE9 and above. Before merging it, I will have to test our products with the update. I'll get back to you, but this is definitely a welcome change, if it works correctly with our software.

**Comment by sunesimonsen** [comment]
Thanks again for the excellent work.

**Comment by sunesimonsen** [comment]
Released as v2.0.0

**Comment by frikeldon** [comment]
You're welcome anytime.

**Pull request merged**